src/ray/raylet/task_dependency_manager.h

```diff
@@ -105,6 +105,11 @@ class TaskDependencyManager {
   /// \return Return a vector of TaskIDs for tasks registered as pending.
   std::vector<TaskID> GetPendingTasks() const;
 
+  /// Cleanup everything for a collection of tasks.
+  ///
+  /// \param driver_task_ids The collection of task IDs for a driver.
```

Comment:
`driver_task_ids` does not match the actual param name.

Reply:
good catch.